File: cpp/LeetCode2942.cpp
class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
        vector<int> ans = {};
        for (int i=0; i<words.size(); i++) {
            string w = words[i];
            for (char c : w) {
                if (c == x) {
                    ans.push_back(i);
                    break;
                }
            }
        }
        return ans;
    }
};

File: python/LeetCode2942.py
class Solution:
    def findWordsContaining(self, words: List[str], x: str) -> List[int]:
        ans = []
        for i in range(len(words)):
            w = words[i]
            for c in w:
                if c == x:
                    ans.append(i)
                    break
        return ans
